doc/classes/FileDialog.xml

```diff
@@ -51,6 +87,30 @@
 				Invalidate and update the current dialog content list.
 			</description>
 		</method>
+		<method name="set_option_default">
+			<return type="void" />
+			<param index="0" name="option" type="int" />
+			<param index="1" name="index" type="int" />
+			<description>
+				Sets default value index of the [OptionButton] or [CheckBox] with index [param option].
+			</description>
+		</method>
+		<method name="set_option_name">
+			<return type="void" />
+			<param index="0" name="option" type="int" />
+			<param index="1" name="name" type="String" />
+			<description>
+				Sets name of the [OptionButton] or [CheckBox] with index [param option].
+			</description>
+		</method>
+		<method name="set_option_values">
+			<return type="void" />
+			<param index="0" name="option" type="int" />
+			<param index="1" name="values" type="PackedStringArray" />
+			<description>
+				Sets array of values of the [OptionButton] with index [param option].
```

Comment:
```suggestion
				Sets the option values of the [OptionButton] with index [param option].
```
This only speaks of `OptionButton`, not `CheckBox` unlike above, should either change?

Reply:
`CheckBox` do not have any values, so this option is relevant only for `OptionBox`. The `name` property is relevant for both.

Reply:
Meant the equivalent entry in `EditorFileDialog`